tools/v8_gypfiles/v8.gyp

```diff
@@ -277,12 +277,6 @@
           # to appear before libv8_snapshot.a so it's listed explicitly.
           'dependencies': ['v8_base', 'v8_snapshot'],
         }],
-        ['v8_use_snapshot==1 and v8_use_external_startup_data==1 and want_separate_host_toolset==0', {
```

Comment:
You can also remove lines 274-279 and remove, change the if/else logic on line 471 to always take the 'else' branch, and drop `v8_use_external_startup_data` from features.gypi (and remove the conditional on line 287 in that file.)

Reply:
Sorry I'd missed this! Fixed now, PTAL.